2023-03-27 10:47:05,711 p=78352 u=teande n=ansible | 192.168.25.172 | FAILED! => {
    "msg": "Using a SSH password instead of a key is not possible because Host Key checking is enabled and sshpass does not support this.  Please add this host's fingerprint to your known_hosts file to manage this host."
}
2023-03-27 10:47:35,464 p=78430 u=teande n=ansible | 192.168.25.172 | UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: Syntax error on line stdin:1 \"/bin/sh -c 'echo ~admin && sleep 0'\": Unknown command",
    "unreachable": true
}
2023-03-27 11:10:49,025 p=81162 u=teande n=ansible | PLAY [all] ******************************************************************************************************************************************************
2023-03-27 11:10:49,032 p=81162 u=teande n=ansible | TASK [Gathering Facts] ******************************************************************************************************************************************
2023-03-27 11:10:49,992 p=81162 u=teande n=ansible | platform_type is set to cisco.fmcansible.fmc
2023-03-27 11:10:50,940 p=81162 u=teande n=ansible | ok: [192.168.25.172]
2023-03-27 11:10:50,948 p=81162 u=teande n=ansible | TASK [Get Domain UUID] ******************************************************************************************************************************************
2023-03-27 11:10:52,780 p=81162 u=teande n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.module_utils.connection.ConnectionError: Failed to download API specification. Status code: 500. Response: b'[Errno 61] Connection refused'
2023-03-27 11:10:52,781 p=81162 u=teande n=ansible | fatal: [192.168.25.172]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/teande/.ansible/tmp/ansible-local-811629qy5vehb/ansible-tmp-1679895651.836046-81203-132905481835092/AnsiballZ_fmc_configuration.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/Users/teande/.ansible/tmp/ansible-local-811629qy5vehb/ansible-tmp-1679895651.836046-81203-132905481835092/AnsiballZ_fmc_configuration.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/teande/.ansible/tmp/ansible-local-811629qy5vehb/ansible-tmp-1679895651.836046-81203-132905481835092/AnsiballZ_fmc_configuration.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.cisco.fmcansible.plugins.modules.fmc_configuration', init_globals=dict(_module_fqn='ansible_collections.cisco.fmcansible.plugins.modules.fmc_configuration', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/modules/fmc_configuration.py\", line 142, in <module>\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/modules/fmc_configuration.py\", line 123, in main\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 259, in execute_operation\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 272, in crud_operation\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 292, in get_operation_spec\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_gqwhce1p/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible/module_utils/connection.py\", line 200, in __rpc__\nansible.module_utils.connection.ConnectionError: Failed to download API specification. Status code: 500. Response: b'[Errno 61] Connection refused'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
2023-03-27 11:10:52,783 p=81162 u=teande n=ansible | PLAY RECAP ******************************************************************************************************************************************************
2023-03-27 11:10:52,783 p=81162 u=teande n=ansible | 192.168.25.172             : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2023-03-27 11:10:52,884 p=81169 u=teande n=ansible | shutdown complete
2023-03-27 11:11:34,019 p=81312 u=teande n=ansible | PLAY [all] ******************************************************************************************************************************************************
2023-03-27 11:11:34,027 p=81312 u=teande n=ansible | TASK [Gathering Facts] ******************************************************************************************************************************************
2023-03-27 11:11:34,964 p=81312 u=teande n=ansible | platform_type is set to cisco.fmcansible.fmc
2023-03-27 11:11:35,893 p=81312 u=teande n=ansible | ok: [192.168.25.172]
2023-03-27 11:11:35,901 p=81312 u=teande n=ansible | TASK [Get Domain UUID] ******************************************************************************************************************************************
2023-03-27 11:11:54,672 p=81312 u=teande n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.module_utils.connection.ConnectionError: Failed to download API specification. Status code: 500. Response: b'The read operation timed out'
2023-03-27 11:11:54,675 p=81312 u=teande n=ansible | fatal: [192.168.25.172]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/teande/.ansible/tmp/ansible-local-813124rt0ky0l/ansible-tmp-1679895696.781734-81354-142280799963495/AnsiballZ_fmc_configuration.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/Users/teande/.ansible/tmp/ansible-local-813124rt0ky0l/ansible-tmp-1679895696.781734-81354-142280799963495/AnsiballZ_fmc_configuration.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/teande/.ansible/tmp/ansible-local-813124rt0ky0l/ansible-tmp-1679895696.781734-81354-142280799963495/AnsiballZ_fmc_configuration.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.cisco.fmcansible.plugins.modules.fmc_configuration', init_globals=dict(_module_fqn='ansible_collections.cisco.fmcansible.plugins.modules.fmc_configuration', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/modules/fmc_configuration.py\", line 142, in <module>\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/modules/fmc_configuration.py\", line 123, in main\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 259, in execute_operation\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 272, in crud_operation\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible_collections/cisco/fmcansible/plugins/module_utils/configuration.py\", line 292, in get_operation_spec\n  File \"/var/folders/nz/s7wv7t015mj0xd92z6s346_00000gn/T/ansible_cisco.fmcansible.fmc_configuration_payload_acj5nke5/ansible_cisco.fmcansible.fmc_configuration_payload.zip/ansible/module_utils/connection.py\", line 200, in __rpc__\nansible.module_utils.connection.ConnectionError: Failed to download API specification. Status code: 500. Response: b'The read operation timed out'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
2023-03-27 11:11:54,680 p=81312 u=teande n=ansible | PLAY RECAP ******************************************************************************************************************************************************
2023-03-27 11:11:54,682 p=81312 u=teande n=ansible | 192.168.25.172             : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2023-03-27 11:11:54,782 p=81320 u=teande n=ansible | shutdown complete
